pkg/api/blackduck/v1/types.go

```diff
@@ -91,13 +94,13 @@ type PVC struct {
 
 // PostgresExternalDBConfig contain the external database configuration
 type PostgresExternalDBConfig struct {
-	PostgresHost          string `json:"postgresHost"`
-	PostgresPort          int    `json:"postgresPort"`
-	PostgresAdmin         string `json:"postgresAdmin"`
-	PostgresUser          string `json:"postgresUser"`
-	PostgresSsl           bool   `json:"postgresSsl"`
-	PostgresAdminPassword string `json:"postgresAdminPassword"`
-	PostgresUserPassword  string `json:"postgresUserPassword"`
+	Host          string `json:"host"`
```

Comment:
This change would completely break the operator for customers who currently use external databases unless we create a new version of the blackduck custom resource. However, I do not think that renaming variables is worth creating a new CRD version.

Reply:
Change reverted.